python/mxnet/gluon/nn/activations.py

```diff
@@ -158,7 +158,8 @@ def __init__(self, alpha=1.0, **kwargs):
         self._alpha = alpha
 
     def hybrid_forward(self, F, x):
-        return F.where(x > 0, x, self._alpha * (F.exp(x) - 1.0))
+        _x = F.where(x < 0, x, F.zeros_like(x))
+        return F.where(x > 0, x, self._alpha * (F.exp(_x) - 1.0))
```

Comment:
Thanks for your contribution!
Actually we do have ELU implemented in the backend so you can call it directly
Please use:
```Python
return F.LeakyReLU(x, act_type='elu', slope=self._alpha)
```
here instead and also you want to change: https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_gluon.py#L1183 to:
```Python
return mx.nd.expm1(x) if x <= 0.0 else x
```
so that the test would still pass after this change.

Reply:
This bug does not affect the forward calculation, but occurs when the gradient is calculated backwards.

Reply:
@fierceX Yes I understand, and I'm presenting the recommended way to fix this issue to you in the above comment.